scripts/globals/chocobo_digging.lua

```diff
@@ -1119,20 +1119,20 @@ xi.chocoboDig.start = function(player, precheck)
             player:setCharVar('[DIG]DigCount', 0) -- Reset player dig count/fatigue.
         end
 
-        if zoneDugCurrent + 1 > DIG_ZONE_LIMIT then
-            if skillRank < 10 and xi.settings.main.DIG_FATIGUE_SKILL_UP then -- Safety check. Let's not try to skill-up if at max skill.
+        if zoneDugCurrent + 1 > digZoneLimit then
+            if skillRank < 10 and xi.settings.main.digFatigue_SKILL_UP then -- Safety check. Let's not try to skill-up if at max skill.
```

Comment:
This looks like a bad replace all as DIG_FATIGUE_SKILL_UP is a setting in the settings file and thus should stay as DIG_FATIGUE_SKILL_UP or it will not work.

Reply:
yep good catch